docs/css-animations.md

```diff
@@ -1,8 +1,52 @@
-> Below is a comment stolen from a controlling position ( https://github.com/Matt-Esch/virtual-dom/issues/104#issuecomment-68611995 ) question.
+Based on a discusison in ( https://github.com/Matt-Esch/virtual-dom/issues/104#issuecomment-68611995 ) question.
 
-You should be activating the CSS transitions using hooks and nextTick. That is, you should know exactly which nodes you wish to animate based on your data, and you should use that data to add a transition hook based on next tick.
+You should be activating the CSS transitions using hooks and nextTick. Here is a basic example of inserting an element through transition:
+```javascript
+Item.render = function(state) {
+  return h('li', {
+    'class' : new ItemInsertHook('item'),
+  }, [
+    h('div.text', state.text), 
+    h('button', {
+      'ev-click' : mercury.event(...)
+    }, 'Remove or something...'),
+  ]);
+}
 
-This is where the flaw in this topic is; the lack of encoding of the animation state. You don't have to do animations with JS, I don't recommed it over CSS transitions, but you do need to model your expectations properly in your data model and apply transitions to the nodes using that data. Generic transitions that rely on the way in which the DOM is mutated isn't going to work consistently.
+function ItemInsertHook(value) {
+  this.value = value;
+}
+
+ItemInsertHook.prototype.hook = function(elem, propName) {
+  if (!elem.childNodes.length) {
```

Comment:
Yes, I agree `if (!elem.childNodes.length) {` is a quite lame way to check if the node is a newly inserted one. 
`Node.contains(elem)`, looks like a much better option.